### ISSUE TITLE
Firefox / Firebug 51.0.1 says unreachable code after return statement at 63:16

### ISSUE THREAD

**Issue by ccoltman** [M1Sh0u/pStrength#2]
I think it's because it will never reach the second return, "return this;"   

```
return this.each($.proxy(function (idx, pStrengthElement) {
                    pStrengthElementsDefaultStyle[$(pStrengthElement)] = {
                        'background': $(pStrengthElement).css('background'),
                        'color': $(pStrengthElement).css('color')
                    }
                    calculatePasswordStrength.call(pStrengthElement);
                    $(pStrengthElement).data("pStrength", true).bind(settings.bind, function(){
                        calculatePasswordStrength.call(this, pStrengthElement); //***
                    });
                }, this));
return this;
```

**Comment by baerengraben** [comment]
I can confirm this issue. Commenting out "return this" solves this Issue.

**Comment by M1Sh0u** [comment]
Fixed in [Release 1.0.10](https://github.com/M1Sh0u/pStrength/releases/tag/1.0.10)
